Card component document (#15755)
abpframework/abp: docs/en/UI/Angular/Card-Component.md

# Card Component

The ABP Card Component is a wrapper component for the Bootstrap card class.

## Usage

> ABP Card Component is a built-in `ThemeSharedModule` component. If you've imported a shared module into your feature module, you don't need to do anything. 

Reviewer: ```suggestion
ABP Card Component is a part of the `ThemeSharedModule` module. If you've imported that module into your module, you don't need to import it again. If not, first import it as shown below:
```

```ts
// my-feature.module.ts

import { ThemeSharedModule } from '@abp/ng.theme.shared';
import { CardDemoComponent } from './chart-demo.component';

@NgModule({
  imports: [

>     SharedModule,

Reviewer 2: ```suggestion
    ThemeSharedModule ,
```

    // ...
  ],
  declarations: [CardDemoComponent],
  // ...
})
export class MyFeatureModule {}

```

Then, the `abp-card` component can be used. See the example below:
```ts

// card-demo.component.ts

import { Component } from '@angular/core';

@Component({
  selector: 'app-card-demo',
  template: ` 
    <abp-card [cardStyle]="{width: '18rem'}">
      <abp-card-body>
        <abp-card-title>Lorem Ipsum</abp-card-title>

>         Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla commodo condimentum ligula, sed varius nibh eleifend sit amet. Maecenas facilisis vel arcu nec maximus.</abp-card-body>

Reviewer: ```suggestion
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla commodo condimentum ligula, sed varius nibh eleifend sit amet. Maecenas facilisis vel arcu nec maximus.
      </abp-card-body>
```
